bin/pubsub_cli.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
This is literally the worst cli I've written. It's mediocre at best. But it works and I didn't want to spend more time on it at this point. I figure we can hone it as needs arise.

Reply:
I think you're being too hard on yourself here. It's at least [premium mediocre](https://www.ribbonfarm.com/2017/08/17/the-premium-mediocre-life-of-maya-millennial/).

Reply:
Ha!

Reply:
Also, why are you reading the self-review pull request comments from a deranged developer on a solo project doing yet another Sisyphusian infrastructure migration? I am concerned for your health.

Reply:
The concern is mutual, my friend. Don’t hesitate to reach out if you’d like to chat.